actor/builtin/miner/miner_test.go

```diff
@@ -935,32 +935,26 @@ func TestVerifyPIP(t *testing.T) {
 	minerAddr := th.CreateTestMiner(t, st, vms, address.TestAddress, origPid)
 
 	sectorId := uint64(1)
-	commD := th.MakeCommitment()
+	pip := []byte{24, 0, 0, 0, 0, 0, 0, 0, 92, 46, 114, 56, 244, 245, 178, 65, 110, 127, 140, 185, 130, 36, 177, 209, 113, 122, 121, 150, 227, 8, 36, 77, 9, 242, 66, 104, 144, 231, 230, 105, 229, 170, 199, 185, 252, 141, 159, 211, 47, 128, 21, 148, 42, 6, 242, 12, 91, 193, 150, 55, 181, 213, 107, 234, 29, 139, 241, 240, 236, 154, 176, 51}
```

Comment:
@laser probably doesn't like this

Reply:
@acruikshank is also working on a fake VMContext precisely so we can test things like this more easily. Consider a new miner_test2.go file that uses that fake setup for this purpose (and eventually we might port all the tests to it).